thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/dashboard/views/tabs/investigate.ftl

```diff
@@ -102,7 +102,7 @@
           <label class="label-medium-semibold">Current Total</label>
         </div>
         <div class="wow-card-body">
-          {{formatDouble currentValue}}
+          {{formatNumber (formatDouble currentValue)}}
         </div>
         {{#unless externalUrls.INGRAPH}}
           <div class="wow-card-footer">
```

Comment:
No, this one is used for styling purposes only so that it looks consistent with the footers below